### DISCUSSION TITLE
pass thru receiver

### DISCUSSION THREAD

**Discussion by sridharn-dev** [open-telemetry/opentelemetry-collector#2667] [Q&A]
How do i extend opentelemetry to  -  receive data in any format and send to a custom exporter that can consume that data. Like the receiver will receive data in any format and pass thru to an exporter that understands this data.

I started off thinking about defining new datatype no-ops, that just passes thru the data which ever format it comes in. It does look like a bizarre idea given the vision of OpenTelemetry. It really helps in opentelemetry adoptions in stages.

**Comment by bogdandrutu** [comment]
Look at this https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/httpforwarder but not sure how useful will be :)
